README.rst

```diff
@@ -368,6 +368,33 @@ Additionally, these statistics are also accessible from  the command line.
 
 .. image:: demo-django-rq-cli-dashboard.gif
 
+Configuring Prometheus
+----------------------
+
+``django_rq`` also provides a Prometheus compatible view, which can be enabled
+by installing ``prometheus_client`` or installing the extra "prometheus-metrics"
+(``pip install django-rq[prometheus-metrics]``). The metrics are exposed at
```

Comment:
How about naming this`django-rq[prometheus]`? I have no strong feelings about this, just throwing this out there as a shorter alternative.